pandas/tests/groupby/test_groupby.py

```diff
@@ -1694,3 +1694,15 @@ def test_groupby_agg_ohlc_non_first():
     result = df.groupby(pd.Grouper(freq='D')).agg(['sum', 'ohlc'])
 
     tm.assert_frame_equal(result, expected)
+
```

Comment:
all of these tests need to go in pandas/tests/groupby/test_categorical.py with the rest of the observed tests